src/filesystem/impls/filer/lib/HTMLRewriter.js

```diff
@@ -116,9 +146,17 @@ define(function (require, exports, module) {
         rewriter.elements('video', 'src');
         rewriter.elements('audio', 'src');
 
+        rewriter.parseScripts('script','src');
```

Comment:
Should be up with other lines, and called `rewriter.scripts()`, you don't need args because scripts are always `script` and `src`